apps/designer/app/designer/features/style-panel/sections/layout/layout.tsx

```diff
@@ -49,6 +45,18 @@ const layoutSectionFlexStyle = {
       },
     },
   },
+  // @todo justifyItems icons are using justifycontent icons atm
+  "& [data-property=justifyItems]": {
```

Comment:
why are you doubling down on nested selectors if we already agreed to migrate to css-vars